### ISSUE TITLE
Keyboard shortcuts sometimes don't work

### ISSUE THREAD

**Issue by sukiletxe** [edrlab/thorium-reader#1197]
I cannot provide steps to reproduce as it doesn't always happen, but specifically ctrl+pgup and ctrl+pgdn sometimes don't jump to next and previous chapters.

**Comment by lpintes** [comment]
I can confirm this. And I would say that not sometimes, but quite often.

**Comment by lpintes** [comment]
Currently I am running the latest version from Git and I am trying to read the following book:

```
Building Microservices with Go
Nic Jackson
Published on 07/27/2017
Publisher: Packt Publishing
```

**Comment by danielweck** [comment]
Thank you for reporting this. Could you please specify your operating system (Windows, Linux or MacOS). Could you also indicate whether or not you are using Thorium with a screen reader (and if so, which one). Most importantly, which version of Thorium are you using? (on GitHub there are the official "stable" releases, and the "nightly" automated builds)

**Comment by lpintes** [comment]
Windows 10, version 2004
I am running NVDA Alpha, updated regularly.
I built Thorium from source, so it is the latest version from master.

**Comment by danielweck** [comment]
Okay thank you. The `master` branch corresponds to the latest stable release, currently deployed via the Windows Store. The `develop` branch contains the most up to date code. To save you building the application yourself, you may downloaded the automatically-generated executable from https://github.com/edrlab/thorium-reader/releases/tag/latest-windows

**Comment by lpintes** [comment]
Oh sorry. I tested against develop branch.
I cloned and tested, and didn't notice that a develop branch is a default one.

**Comment by m-c-g** [comment]
I've found a reproducible case.
This is on a Mac, Mojave, with the compiled reader version from the current DMG.

Open a document in full screen, accidentally hit ctrl-s, the arrow keys no longer work. 

You have to leave full screen (why doesn't esc work for this?) and realize you have
opened the settings panel and close it.
 
If there are modalities in the interface you need visual feedback indicating them and 
how to get back where you were.

The only documentation for the keyboard functions I find is the settings panel from
the main screen, which you can't find from the reader screen. It doesn't give any
explanation of how the interface works other than the command that gets issued.
Additionally, the focus search and focus tool-bar keystrokes don't appear to function.

Thorium renders documents beautifully. With a better interface and documentation it
could easily become a multiplatform standard.

**Comment by danielweck** [comment]
Great report, thank you.

**Comment by sukiletxe** [comment]
However that's not always the case. Sometimes they don't work, you insist and then they do work. Happened on stable, happens also on Nightly. Windows 10, NVDA screen reader.

**Comment by danielweck** [comment]
Comment from NNELS (Danny Faris @BCLibCoop ):

> We have noticed that Thorium will ocasionally become unresponsive after switching away from the window to another application, then switching back. This only seems to be an issue with the keyboard and screen reader, visually the interface continues to work fine with the mouse.

**Comment by gautierchomel** [comment]
I reproduce the error with NVDA and windows 10 : when on fullscreen `CTRL + Page Down` does not seem to work.
However this does not affects all books. An [old epub2](https://www.010101book.net/fr/epub/le_livre_010101_1971_2015_marie_lebert.epub) is affected but not a recent epub3 (searching for a public reference). 
The difference between both files is that the second one has `<section>`.

**Comment by gautierchomel** [comment]
Using `CTRL + ALT + SHIFT + left or right arrow` instead of `CTRL+Page down or Up`. It works in all cases.
